lib/kitchen/directions/bake_chapter_summary.rb

```diff
@@ -27,6 +27,9 @@ def bake(chapter:, metadata_source:, uuid_prefix: '.', klass: 'summary')
           # TODO: include specific page types somehow without writing it out
           chapter.non_introduction_pages.each do |page|
             summary = page.summary
+
+            next if summary.nil?
```

Comment:
this works, but you could also use `next unless summary` to check for truthiness